File: unittest/scripts/auto/py_devapi/scripts/mysqlx_collectionadd_help.py
shell.connect(__uripwd)
schema = session.create_schema('collectionadd_help')
coll = schema.create_collection('sample')
colladd = coll.add({'name':'test'});
session.close()

#@ colladd
colladd.help()

#@ global ? for CollectionAdd[USE:colladd]
\? CollectionAdd

#@ global help for CollectionAdd[USE:colladd]
\help CollectionAdd

#@ colladd.add
colladd.help('add')

#@ global ? for add[USE:colladd.add]
\? CollectionAdd.add

#@ global help for add[USE:colladd.add]
\help CollectionAdd.add

#@ colladd.execute
colladd.help('execute')

#@ global ? for execute[USE:colladd.execute]
\? CollectionAdd.execute

#@ global help for execute[USE:colladd.execute]
\help CollectionAdd.execute

#@ colladd.help
colladd.help('help')

#@ global ? for help[USE:colladd.help]
\? CollectionAdd.help

#@ global help for help[USE:colladd.help]
\help CollectionAdd.help


shell.connect(__uripwd)
session.drop_schema('collectionadd_help');
session.close()

File: unittest/scripts/auto/js_devapi/scripts/mysqlx_collectionadd_help.js
//@ Initialization
shell.connect(__uripwd);
var schema = session.createSchema('collectionadd_help');
var collection = schema.createCollection('sample');
var crud = collection.add({one:1})
session.close();

//@ CollectionAdd help
crud.help();

//@ CollectionAdd help, \? [USE:CollectionAdd help]
\? CollectionAdd

//@ Help on add
crud.help('add');

//@ Help on add, \? [USE:Help on add]
\? CollectionAdd.add

//@ Help on execute
crud.help('execute');

//@ Help on execute, \? [USE:Help on execute]
\? CollectionAdd.execute

//@ Help on help
crud.help('help');

//@ Help on help, \? [USE:Help on help]
\? CollectionAdd.help

//@ Finalization
shell.connect(__uripwd);
session.dropSchema('collectionadd_help');
session.close();
